helpers/calendar/events.py

```diff
@@ -90,3 +92,25 @@ def get_all_room_schedules(self, query, start_date, end_date):
                 all_events.append(current_event)
 
         return all_events, all_dates
+
+    def check_event_status(self, info, **kwargs):
+        try:
+            room_id = RoomModel.query.filter_by(
```

Comment:
Great work @C3real-kill3r.

Kindly share the reason for using .first().
Rooms are to have unique calender_ids so the data returned here is not likely to be more than one.

Reply:
Yes, rooms have unique ids, but without the .first() I'll get an attribute error since the query wont have the attribute id. So to bypass that I added the function. @RonKbS